lib/schema_web/controllers/schema_controller.ex

```diff
@@ -122,6 +135,45 @@ defmodule SchemaWeb.SchemaController do
     send_json_resp(conn, version)
   end
 
+  @doc """
+  Get available OCSF schema versions.
+  """
+  swagger_path :versions do
+    get("/api/versions")
+    summary("Versions")
+    description("Get available OCSF schema versions.")
+    produces("application/json")
+    tag("Schema")
+    response(200, "Success", :Versions)
+  end
+
+  @spec versions(Plug.Conn.t(), any) :: Plug.Conn.t()
+  def versions(conn, _params) do
+
+    url = Application.get_env(:schema_server, SchemaWeb.Endpoint)[:url]
+
+    base_url = "#{conn.scheme}://#{Keyword.fetch!(url, :host)}:#{Keyword.fetch!(url, :port)}"
```

Comment:
```suggestion
    # The :url key is meant to be set for production, but isn't set for local development
    base_url = if url == nil do
      "#{conn.scheme}://#{conn.host}:#{conn.port}"
    else
      "#{conn.scheme}://#{Keyword.fetch!(url, :host)}:#{Keyword.fetch!(url, :port)}"
    end
```
This change allows the `versions` API to work with a local configuration, when the `url` configuration isn't present.

Reply:
I agree that allowing it to work with a local configuration is a goal. But in my local development, it seems like `url` is set? I don't know but it seems like there are some default parameters that are always present, so `url` is never nil, rendering this ineffective. I was looking here [url](https://github.com/ocsf/ocsf-server/blob/a74a4b7c8d06af7c55de65d4e741479d943995d7/config/releases.exs#L25) but I'm not sure whether that's the right place.

Reply:
I hit this when running locally (without Docker) using this command (from the `README.md`):
```shell
SCHEMA_DIR=../ocsf-schema SCHEMA_EXTENSION=extensions iex -S mix phx.server
```

Reply:
I see, that's probably where the discrepancy is coming from - I'm using the dockerized env for testing